frontend/src/metabase/components/MetadataInfo/ColumnInfoIcon/ColumnInfoIcon.tsx

```diff
@@ -10,19 +10,28 @@ import {
   QueryColumnInfoPopover,
   TableColumnInfoPopover,
 } from "../ColumnInfoPopover";
-import { PopoverHoverTarget, HoverParent } from "../InfoIcon";
+import { PopoverHoverTarget, HoverParent, defaultDelay } from "../InfoIcon";
 export { HoverParent };
 
+export type QueryColumnInfoIconProps = QueryColumnInfoPopoverProps & {
+  showIfEmpty?: boolean;
+};
+
 export function QueryColumnInfoIcon({
   className,
-  delay = [0, 150],
+  delay = defaultDelay,
+  showIfEmpty = false,
   ...props
-}: QueryColumnInfoPopoverProps) {
+}: QueryColumnInfoIconProps) {
   const { query, stageIndex, column } = props;
   const { description = "" } = query
     ? Lib.displayInfo(query, stageIndex, column)
     : {};
 
+  if (!description && !showIfEmpty) {
+    return null;
```

Comment:
Suggestions without info icon now have different height than the ones with the icon.
I think they should have the same size.

In the screenshot the "Total" column does not have an icon and the background color has been added with developer tools to highlight the difference in height.


![image](https://github.com/metabase/metabase/assets/6830683/ed1a5127-fe94-4e29-9bf9-17a2acca8a8a)

Reply:
Good catch. I've updated some other components as well.

Reply:
Now rows without icons are 24px high, and the rows with icons are 26px high.
Please fix it before merging :pray: